lib/transbank/sdk/patpass/patpass_by_webpay/transaction.rb

```diff
@@ -0,0 +1,76 @@
+module Transbank
+  module Patpass
+    module PatpassByWebpay
+      class Transaction
+        extend Transbank::Utils::NetHelper
+        CREATE_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+        COMMIT_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+        TRANSACTION_STATUS_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+
+        class << self
+
+          def create(buy_order:, session_id:, amount:, return_url:, details: ,options: nil)
```

Comment:
In java the create receive all the wpm_details as a param https://github.com/TransbankDevelopers/transbank-sdk-java/blob/66ad04ca2f970e82e567b426bb8e1d2bf2feb582/src/main/java/cl/transbank/patpass/PatpassByWebpay.java#L79, this should be the same for consistency in operation.